security/middleware.py

```diff
@@ -692,8 +694,14 @@ def process_response(self, request, response):
         enforcement or report-only headers in all currently used variants.
         """
         # choose headers based enforcement mode
+        is_ie = False
+        if 'HTTP_USER_AGENT' in request.META:
+            parsed_ua = ParseUserAgent(request.META['HTTP_USER_AGENT'])
+            is_ie = parsed_ua['family'] == 'IE'
+
         if self._enforce:
-            header = 'Content-Security-Policy'
+            header = 'X-Content-Security-Policy' if is_ie \
```

Comment:
Don't use `\` for line continuation. It's better to use parentheses. (\ can be finicky. A space after the \ breaks things, for example.) I'd just write this as:

```
if is_ie:
    header = 'X-Content-Security-Policy'
else:
    header = 'Content-Security-Policy'
```